IpmiFeaturePkg/Include/IpmiFeature.h

```diff
@@ -0,0 +1,64 @@
+/** @file
+  Ipmi structure defintions that are either not defined in MdePkg's Ipmi header files
+  or are extensions of existing function headers.
+
+  Copyright (c) Microsoft Corporation
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef IPMI_FEATURE_H__
+#define IPMI_FEATURE_H__
+
+#include <IndustryStandard/Ipmi.h>
+
+// Defintion for IPMI_APP_GET_SYSTEM_GUID
+typedef struct {
+  UINT8    CompletionCode;
+  UINT8    Guid[16];
+} IPMI_GET_SYSTEM_GUID_RESPONSE;
+
+// Definitions for IPMI_APP_GET_SYSTEM_INTERFACE_CAPABILITIES

```

Comment:
The names in the comments are different than the structure names, perhaps that is intentional, pointing to other code that call them the comment name?

Reply:
I see them below.